acrobot.py

```diff
@@ -10,18 +10,27 @@
 from pydrake.all import *
 from ilqr import IterativeLinearQuadraticRegulator
 import time
+import utils_derivs_interpolation
 
 ####################################
 # Parameters
 ####################################
 
-T = 1.5        # total simulation time (S)
-dt = 1e-2      # simulation timestep
+T = 3           # total simulation time (S)
+dt = 0.004      # simulation timestep
+
+# Parameters for derivative interpolation
```

Comment:
In all of these, I'd like `None` to be an option, and for `None` to be selected by default

Reply:
Hi Vince, There is this kind of idea already in the code, if None is passed into the ilqr algorithm then it uses the baseline case of calculating derivatives at every time step (which can be thought of as SetInterval1). I have added a variable above the interpolation settings that is currently set to false so that no interpolation is used unless they select true, will this do what you wanted?

Reply:
Yup, that'll do what I was looking for! I just wanted an explicit switch for that behavior